packages/teleport/src/services/user/makeUser.ts

```diff
@@ -21,7 +21,7 @@ export default function makeUser(json): User {
   const [name, roles, authType] = at(json, ['name', 'roles', 'authType']);
   return {
     name,
-    roles: roles || [],
+    roles: roles ? roles.sort() : [],
```

Comment:
Should this use the same sort method as above? Looking at the snapshot it looks like the sort is now out of order (unless I'm reading it wrong).

Reply:
You're right the sort is now out of order b/c in storybook we mock the users and it doesn't go through the `makeUser` that sorts it, which i think it's fine. I moved the sorting here so we are not sorting the list of roles every time the table re-renders (and I needed sorted list to sort the column).